Firestore/Example/FuzzTests/FuzzingTargets/FSTFuzzTestSerializer.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIRESTORE_EXAMPLE_FUZZTESTS_FUZZINGTARGETS_FSTFUZZTESTSERIALIZER_H_
+#define FIRESTORE_EXAMPLE_FUZZTESTS_FUZZINGTARGETS_FSTFUZZTESTSERIALIZER_H_
+
+#import <Foundation/Foundation.h>
+
+namespace firebase {
+namespace firestore {
+namespace fuzzing {
+
+// Returns the location of the Serializer dictionary file.
+inline NSString *GetSerializerDictionaryLocation(NSString *resources_location) {
+  return [resources_location stringByAppendingPathComponent:@"Serializer/serializer.dictionary"];
+}
+
+// Returns the location of the Serializer corpus. This corpus is a special
+// case because we generate its binary protos during the build process.
+inline NSString *GetSerializerCorpusLocation() {
+  return @"FuzzTestsCorpus";
+}
+
+// Fuzz-test the deserialization process in Firestore. The Serializer reads raw
+// bytes and converts them to a model object.
+int FuzzTestDeserialization(const uint8_t *data, size_t size);
```

Comment:
nit: Technically, you should include <cstdint> in order to bring uint8_t into scope. (IWYU)

Reply:
So should I include `stdint.h` and  `cstddef`?

Reply:
Github mangled my comment a bit. (It strips http tags.) So lets try again, this time in `code font` so that the angle brackets (and their contents) don't get removed. :)

... include `<cstdint>` ...

Re cstdint vs stdint.h: It doesn't much matter, but we've decided to go toward cfoo instead of foo.h for c includes.

Re cstddef: Good point. I guess you technically need this for size_t.

So summary: Add
```
#include <cstddef>
#include <cstdint>
```
to the top of the file.

Reply:
Thanks. Done, pushed, and merged.